tests/requirements.txt

```diff
@@ -0,0 +1,14 @@
+atomicwrites==1.1.5
+attrs==18.1.0
+boto3==1.7.64
+botocore==1.10.64
+docutils==0.14
+jmespath==0.9.3
+more-itertools==4.2.0
+pluggy==0.6.0
+py==1.5.4
+pytest==3.6.3
+python-dateutil==2.7.3
+s3transfer==0.1.13
+six==1.11.0
+cfn-lint
```

Comment:
Should we not set a version here? 
otherwise if it updates - there could be something that breaks

Reply:
pytest is an infrastructure to run tests written in python. We can use it for testing our AWS templates and maybe Azure as well in the future. You are right regarding the version that is missing. I'll add it

Reply:
Fixed